src/pages/login.tsx

```diff
@@ -159,6 +160,15 @@ export default function Login() {
       });
     }
 
+    const { data: authData, error: authError } =
+      await supabase.auth.signInAnonymously();
```

Comment:
How does this work? How does supabase know what permission levels the user has? Doesn't seem like we are providing a token or anything

Reply:
supabase handles it by storing in localStorage. we don't need any specific permissions to do PSI stuff